africanlii/templates/peachjam/_footer.html

```diff
@@ -56,6 +56,9 @@ <h5>{% trans 'About' %} {{ APP_NAME }}</h5>
   <div>
     <a href="https://africanlii.org/" target="_blank" rel="noreferrer">AfricanLII</a>
   </div>
+  <div>
```

Comment:
Let's keep CIVLII as a temporary staging area for now, and not link it into the network.